Documentation/Helm-Charts/ceph-cluster-chart.md.gotmpl

```diff
@@ -0,0 +1,156 @@
+
+Rook currently publishes builds of this chart to the `release` and `master` channels.
+
+**Before installing, review the values.yaml to confirm if the default settings need to be updated.**
+
+* If the operator was installed in a namespace other than `rook-ceph`, the namespace
+  must be set in the `operatorNamespace` variable.
+* Set the desired settings in the `cephClusterSpec`. The [defaults](https://github.com/rook/rook/tree/master/deploy/charts/rook-ceph-cluster/values.yaml)
+  are only an example and not likely to apply to your cluster.
+* The `monitoring` section should be removed from the `cephClusterSpec`, as it is specified separately in the helm settings.
+* The default values for `cephBlockPools`, `cephFileSystems`, and `CephObjectStores` will create one of each, and their corresponding storage classes.
+* All Ceph components now have default values for the pod resources. The resources may need to be adjusted in production clusters depending on the load. The resources can also be disabled if Ceph should not be limited (e.g. test clusters).
+
+### **Release**
+
+The release channel is the most recent release of Rook that is considered stable for the community.
+
+The example install assumes you have first installed the [Rook Operator Helm Chart](operator-chart.md)
+and created your customized values-override.yaml.
+
+```console
+helm repo add rook-release https://charts.rook.io/release
+helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
+   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values-override.yaml
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the rook-operator chart and their default values.
+
+{{ template "chart.valuesTable" . }}
+
+### **Ceph Cluster Spec**
+
+The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
+For the full list, see the [Cluster CRD](../CRDs/Cluster/ceph-cluster-crd.md) topic.
+
+### **Ceph Block Pools**
+
+The `cephBlockPools` array in the values file will define a list of CephBlockPool as described in the table below.
+
+| Parameter                           | Description                                                                                                                                                                                                          | Default          |
+| ----------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| `name`                              | The name of the CephBlockPool                                                                                                                                                                                        | `ceph-blockpool` |
```

Comment:
For consistency with the generated table, how about removing the extra spaces? It will be much better for diffs if a long setting name doesn't cause all the whitespace to change, which has been an annoying behavior for a long time.

Reply:
We can also diable the table formatter by adding this to .vscode/settings.json:
```
"markdown.extension.tableFormatter.enabled": false,
```